index.bs

```diff
@@ -1869,13 +1869,13 @@ If an attribute has no <emu-t>static</emu-t> keyword, then it declares a
 it declares a [=static attribute=]. Note that in addition to being [=interface members=],
 [=read only=] [=regular attributes=] can be [=namespace members=] as well.
 
+The <dfn>getter steps</dfn> of an attribute |attr| should be introduced using text of the form “The
+<code>|attr|</code> getter steps are:” followed by a list, or “The <code>|attr|</code> getter steps
```

Comment:
I believe we use straight quotes in this document…

Reply:
I could only find quotes used around code constructs, not natural language. I could go either way here.